projects/security.md

```diff
@@ -0,0 +1,79 @@
+    * [`Code signature`](https://www.elastic.co/guide/en/ecs/current/ecs-code_signature.html)
+    * [`DLL`](https://www.elastic.co/guide/en/ecs/current/ecs-dll.html)
+    * [`DNS`](https://www.elastic.co/guide/en/ecs/current/ecs-dns.html)
+    * [`File`](https://www.elastic.co/guide/en/ecs/current/ecs-file.html)
+    * [`Group`](https://www.elastic.co/guide/en/ecs/current/ecs-group.html)
+    * [`Hash`](https://www.elastic.co/guide/en/ecs/current/ecs-hash.html)
+    * [`Host`](https://www.elastic.co/guide/en/ecs/current/ecs-host.html)
+    * [`Network`](https://www.elastic.co/guide/en/ecs/current/ecs-network.html)
+    * [`Operating System`](https://www.elastic.co/guide/en/ecs/current/ecs-os.html)
+    * [`Package`](https://www.elastic.co/guide/en/ecs/current/ecs-package.html)
+    * [`Process`](https://www.elastic.co/guide/en/ecs/current/ecs-process.html)
+    * [`Registry`](https://www.elastic.co/guide/en/ecs/current/ecs-registry.html)
+    * [`Risk information`](https://www.elastic.co/guide/en/ecs/current/ecs-risk.html)
+    * [`Rule`](https://www.elastic.co/guide/en/ecs/current/ecs-rule.html)
+    * [`Threat`](https://www.elastic.co/guide/en/ecs/current/ecs-threat.html)
+    * [`TLS`](https://www.elastic.co/guide/en/ecs/current/ecs-tls.html)
+    * [`User`](https://www.elastic.co/guide/en/ecs/current/ecs-user.html)
+    * [`Vulnerability`](https://www.elastic.co/guide/en/ecs/current/ecs-vulnerability.html)
+  * Please note that some of the above-mentioned namespaces are already a part of the Semantic Conventions schema. The goal is to expand these namespaces to include additional fields that are relevant to security use cases.
+
+* As new use cases and namespaces are introduced to the semantic conventions, there may be a need for additional instrumentation to accommodate them. It is anticipated that this aspect will expand through an iterative process.
+
+## Staffing / Help Wanted
+
+We are seeking security experts to collaborate with us in expanding the security domain within the community.  
+
+### Required staffing
+
+There is an open [PR](https://github.com/open-telemetry/semantic-conventions/issues/580) to create a `semconv-security-approver` group for all PRs related to security fields.  
+
+* project lead: @trisch-me (Elastic)
+* domain expert: @mjwolf (Elastic)
+* domain expert: @raesene (Datadog)
+* domain expert: @lambdanis (Isovalent)
+
+* TC sponsor: @reyang
+* TC sponsor: vacant
+
+Need more 
+- [ ] domain experts
+- [ ] TC 
+- [ ] potentially, maintainers of language-specific instrumentation may be needed if the need arises.
+
+
+## Meeting Times
+
+There is an open slot in the Semantic Conventions WG for this project.
+- Mondays at 8 AM PST
+
+## Timeline
+
+The goal is to have the security semantic conventions implemented by the end of 2024.
+
+The timeline for this project is as follows:
+December 2023: Initial Draft
+April 2024: Review and Refinement
+Mai 2024-December 2024: Introducing the Security Semantic Conventions
```

Comment:
May need to review these dates now that we are already in April.

Reply:
I think dates are correct. I have started to work on it last year